src/content/docs/apm/agents/nodejs-agent/installation-configuration/install-nodejs-agent-docker.mdx

```diff
@@ -27,17 +29,19 @@ With just a few additions your existing Dockerfile can be used with our Node.js
    "newrelic": "latest",
```

Comment:
Is there a space between the 1 bullet and the snippet?

Reply:
There should be a line break, so the snippet gets rendered as a code block.